.github/workflows/promote.yaml

```diff
@@ -10,6 +10,12 @@ jobs:
     uses: ./.github/workflows/build.yaml
     secrets: inherit
 
+  generate-sbom:
+    needs:
+      - build
+    uses: ./.github/workflows/generate-sbom.yaml
+    secrets: inherit
+
   promote-artifacts:
     needs:
       - build
```

Comment:
I think we want to add needs for generate-sbom here, so that we sure it's in the promoted artifact